hadoop-hdds/client/src/main/java/org/apache/hadoop/hdds/scm/XceiverClientManager.java

```diff
@@ -345,10 +337,30 @@ public void setMaxSize(int maxSize) {
       this.maxSize = maxSize;
     }
 
+  }
+
+  /**
+   * Configuration for ratis client.
+   */
+  @ConfigGroup(prefix = "dfs.ratis.client")
+  public class DFSRatisClientConfig {
+
+    @Config(key = "async.max.outstanding.requests",
```

Comment:
I dont think we have a corresponding config in ozone-default.xml.

Reply:
With new-style Java config, we don't need to add, it generates an ozone-default-generated.xml. In that file, I can see this new config.

```
[root@ozone-test-1 ex]# cat ozone-default-generated.xml | grep -A4 "dfs.ratis.client.async"
    <name>dfs.ratis.client.async.max.outstanding.requests</name>
    <value>64</value>
    <description>Controls the maximum number of outstanding async requests that can be handled by the Standalone as well as Ratis client.</description>
    <tag>OZONE, CLIENT, PERFORMANCE</tag>
  </property>
```